### PR TITLE
Update GitHub members section

### PR DESCRIPTION
**Pull request by 0xLeif**

### CODE REVIEW

File: Content/github.md [hunk 1]
```diff
@@ -9,24 +9,11 @@
...
-  </tr>
-</table>
+
+<!-- members -->
+<a href="https://github.com/parshav"><img src="https://github.com/parshav.png" width="60px" alt="Parshav" />
+<a href="https://github.com/0xLeif"><img src="https://github.com/0xLeif.png" width="60px" alt="0xLeif" />
+<a href="https://github.com/haIIux"><img src="https://github.com/haIIux.png" width="60px" alt="haIIux" />
+<a href="https://github.com/rjp2525"><img src="https://github.com/rjp2525.png" width="60px" alt="Reno" />
+<a href="https://github.com/FullQueueDeveloper"><img src="https://github.com/FullQueueDeveloper.png" width="60px" alt="Full Queue Developer" />
+<!-- members -->
```
Comment (haIIux) [inline Content/github.md]: Only complaint as discussed in the Discord (#blog) is that they're not symetrical. Can be something we can consider in the future at a later update.

**Review by haIIux — DISMISSED** [review]

File: Content/github.md (hunk 1 above)
Comment (0xLeif) [reply Content/github.md]: Fixed in https://github.com/0xOpenBytes/Site/pull/20/commits/ae6b72fe1e3e693e0bedac70d468ac9f1862b751

**Review by haIIux — APPROVED** [review]
verry niceeee

**Review by rjp2525 — APPROVED** [review]
Visual QA looks good

**Pull request merged**